.changelog/13244.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:feature
+xds: Consul servers can now handle xDS streams for arbitrary proxy services in the catalog
```

Comment:
@mkeeler I wasn't sure whether to hold off on the changelog entry until the feature is production-ready. What do you think?

Reply:
I would either hold off on the changelog or switch this to be an `improvement` instead of `feature`. That way for people viewing the changelog don't see "new feature" and think its ready for general use.

Reply:
Yeah, let's hold off to avoid confusion 🙇🏻